nbdev/export.py

```diff
@@ -9,6 +9,7 @@
 from .imports import *
 from fastscript import *
 from keyword import iskeyword
+from .flags import parse_line
```

Comment:
Importing a notebook that is numbered after is dangerous, could you move that function either in this notebook or in imports?

Reply:
I think this is safe because flags.py is written the old fashioned way (i.e. not exported from 08_flags.ipynb).
Let me know if I'm missing something and I'll move parse_line to imports.

Reply:
Oh, had missed that. I think the notebook should be renamed flag_tests, to make this clearer.

Reply:
done